src/TypeSpec.Extension/Emitter.Csharp/src/options.ts

```diff
@@ -27,6 +27,7 @@ export interface AzureCSharpEmitterOptions extends CSharpEmitterOptions {
     "use-model-reader-writer"?: boolean;
     "library-name"?: string;
     "examples-dir"?: string;
+    "enable-bicep-serialization": boolean;
```

Comment:
this should be optional
```suggestion
    "enable-bicep-serialization"?: boolean;
```